### PR TITLE
zh-trans: add docs/getting-started-guides/ubuntu/networking.md

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+--- -->
+
+{{% capture overview %}}
+
+<!-- Kubernetes supports the [Container Network Interface (CNI)](https://github.com/containernetworking/cni).
+This is a network plugin architecture that allows you to use whatever
+Kubernetes-friendly SDN you want. Currently this means support for Flannel and Canal. -->
+
+Kubernetes 支持[容器网络接口](https://github.com/containernetworking/cni)。
+这个网络插件架构允许你使用任何你喜欢的、Kubernetes 友好的 SDN。
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
这个网络插件架构允许你使用任何你喜欢的、对 Kubernetes 友好的 SDN。
```
感觉加个对要好一点

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+现在的插件支持是 Flannel 和 Canal。
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
目前支持的插件是 Flannel 和 Canal。
```

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+{{% /capture %}}
+
+
+{{% capture steps %}}
+
+<!-- The CNI charms are [subordinates](https://jujucharms.com/docs/stable/authors-subordinate-applications).
+These charms will require a principal charm that implements the `kubernetes-cni` interface in order to properly deploy. -->
+
+CNI charms 在[子路径](https://jujucharms.com/docs/stable/authors-subordinate-applications)下。
+这些 charms 需要主 charm 实现 `kubernetes-cni` 借口，才能正常部署。
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
这些 charms 需要主 charm 实现 `kubernetes-cni` 接口，才能正常部署。
```

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+<!-- ### Configuration -->
+### 配置
+
+<!-- **iface** The interface to configure the flannel or canal SDN binding. If this value is
+empty string or undefined the code will attempt to find the default network
+adapter similar to the following command: -->
+
+**iface** 借口是用来配置 flannel 或 canal 的 SDN 绑定。
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
**iface** 接口是用来配置 flannel 或 canal 的 SDN 绑定。
```

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+<!-- **cidr** The network range to configure the flannel or canal SDN to declare when
+establishing networking setup with etcd. Ensure this network range is not active
+on layers 2/3 you're deploying to, as it will cause collisions and odd behavior
+if care is not taken when selecting a good CIDR range to assign to flannel. It's
+also good practice to ensure you allot yourself a large enough IP range to support
+how large your cluster will potentially scale.  Class A IP ranges with /24 are
+a good option. -->
+
+**cidr** 在用 etcd 进行网络设置时，需要声明、所要用于配置 flannel 或 canal SDN 的网络地址范围。
+确保这个网络地址范围在所要部署的 L2/L3 上不是有效状态，
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
请确保这个网络地址范围在所要部署的 L2/L3 上不是有效状态，
```
加个'请'，读起来顺畅些

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+否则如果选择的分配给 flannel 的 CIDR 不好的话，就会出现冲突和奇怪的行为。
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
因为如果没有选择一个好的 CIDR 范围来分配给 flannel，就会出现冲突或异常行为。
```

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+同时也要保证 IP 地址范围足够大，可以支持以后可能会发生的集群扩容。
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
同时也要保证 IP 地址范围足够大以支持未来可能会发生的集群扩容。
```
只是建议

File: content/zh/docs/getting-started-guides/ubuntu/networking.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/getting-started-guides/ubuntu/networking.md]: 👍

File: content/zh/docs/getting-started-guides/ubuntu/networking.md (hunk 5 above)
Comment (chenrui333) [reply content/zh/docs/getting-started-guides/ubuntu/networking.md]: 👍

**Comment by chenrui333** [comment]
@Mr-Linus thanks for your suggestions, applied all of them! 👍

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+<!-- **cidr** The network range to configure the flannel or canal SDN to declare when
+establishing networking setup with etcd. Ensure this network range is not active
+on layers 2/3 you're deploying to, as it will cause collisions and odd behavior
+if care is not taken when selecting a good CIDR range to assign to flannel. It's
+also good practice to ensure you allot yourself a large enough IP range to support
+how large your cluster will potentially scale.  Class A IP ranges with /24 are
+a good option. -->
+
+**cidr** 在用 etcd 进行网络设置时，需要声明、所要用于配置 flannel 或 canal SDN 的网络地址范围。
```
Comment (tengqm) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
**cidr** 在用 etcd 进行网络设置时，用于配置 flannel 或 canal SDN 所要使用的网络地址范围。
```

File: content/zh/docs/getting-started-guides/ubuntu/networking.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+请确保这个网络地址范围在所要部署的 L2/L3 上不是有效状态，
```
Comment (AdamDang) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]: “有效状态”是不是改成“启用状态”或者“在用状态” 更好些？
这里关心更多的应该是 IP 段是否被占用。

Comment (chenrui333) [reply content/zh/docs/getting-started-guides/ubuntu/networking.md]: 👍 在用状态 挺好的。

Comment (chenrui333) [inline content/zh/docs/getting-started-guides/ubuntu/networking.md]:
```suggestion
请确保这个网络地址范围在所要部署的 L2/L3 上不是在用状态，
```

**Comment by chenrui333** [comment]
@tengqm we need `/lgtm` as well, it got removed per new change

**Comment by idealhack** [comment]
I haven't take a close look but this looks ready to get in, so...

/lgtm

**Comment by chenrui333** [comment]
thanks @idealhack

**Pull request merged**